arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -1586,12 +1699,12 @@
       "x-ms-azure-resource": true,
       "description": "The Resource definition for other than namespace."
     },
-    "NamespaceListResult": {
+    "SBNamespaceListResult": {
```

Comment:
Per the meeting with @salameer yesterday these breaking changes will be reverted for now.